WebRoot/css/zm.css

```diff
@@ -2166,6 +2166,123 @@ div.CompDragArea * {
 	overflow:hidden;
 }
 
+/* Task View style */
+.ZmTaskInfoView {
+	@TaskView@
+}
+
+.ZmReadingPaneBottom .ZmTaskInfoView {
+	@TaskViewBottomPane@
+}
+
+.ZmTaskInfoView .ZmTaskWrapper {
+	@TaskWrapper@
+}
+
+.ZmTaskInfoView .Object {
+	@Anchor@ 
+}
+
+.ZmTaskInfoView .Object-hover {
+	@Anchor-hover@
+}
+
+/* Task view header style */
+.ZmTaskHeader {
+	@TaskHeader@
+}
+
+.ZmTaskHeader > TABLE {
+	@FullWidth@
+}
+
+.ZmTaskHeader .ZmTaskHeaderRow {
+	@TaskHeaderRow@
+}
+
+.ZmTaskHeaderRow > DIV {
+	@TaskHeaderCell@
+}
+
+.ZmTaskHeaderRow .ZmTaskHeaderMain {
+	@TaskHeaderMain@
+}
+
+.ZmTaskHeaderMain .ZmTaskHeaderImage {
+	@TaskHeaderImageWrapper@
+}
+
+.ZmTaskHeaderImage IMG,
+.ZmTaskHeaderImage .svg-icon,
+.ZmTaskHeaderImage .progress-circle {
+	@TaskHeaderImage@
+}
+
+.ZmTaskHeaderMain .ZmTaskHeaderInfo {
+	@TaskHeaderInfo@
+}
+
+.ZmTaskHeaderInfo .ZmTaskTitle:not(:empty) {
+	@TaskHeaderTitle@
+}
+
+.ZmTaskHeaderInfo .ZmTaskHeaderTags:not(:empty) {
+	@TaskHeaderTags@
+}
+
+.ZmTaskHeaderRow .ZmTaskHeaderMeta {
+	@TaskHeaderMeta@
+}
+
+.ZmTaskHeaderMeta > TABLE {
+	@TaskHeaderMetaTable@
+}
+
+.ZmTaskHeaderMeta .ZmTaskHeaderLocationIcon,
+.ZmTaskHeaderMeta .ZmTaskHeaderPriorityIcon {
+	@TaskHeaderMetaTableIcon@
+}
+
+.ZmTaskHeaderMeta .ZmTaskHeaderLocationName,
+.ZmTaskHeaderMeta .ZmTaskHeaderPriorityName {
+	@TaskHeaderMetaTableValue@
+}
+
+/* Task info/exception style */
+.ZmTaskInfoView .ZmTaskInfo:not(:empty) {
+	@ContactInfo@
```

Comment:
Added: https://github.com/Zimbra/zm-web-client/pull/63/commits/a7b2fcdc7e6d460347cdbfc70cafc577ab173260#diff-33f9c57f22cd743824b2918132c5d73bR2252
and https://github.com/Zimbra/zm-web-client/pull/63/commits/a7b2fcdc7e6d460347cdbfc70cafc577ab173260#diff-16ae02866e4f63813de9a7825a1057e3R1459

Thanks